app/src/main/res/values/dimens.xml

```diff
@@ -99,4 +99,7 @@
     <dimen name="scan_progress_margin_top">10dp</dimen>
     <dimen name="device_setup_margin_top">20dp</dimen>
 
+    <dimen name="search_skill_padding">7dp</dimen>
+    <dimen name="search_skill_margin">10dp</dimen>
```

Comment:
Repeated margin value. Use margin_large already present in the file.

Reply:
@arundhati24 it is the name that matters so that the code is easily readable

Reply:
@batbrain7 There is another issue that I had opened that will be removing repeated dimens so as to use standard padding/margin values and naming conventions as per the Material Design Guidelines for a more consistent design. Hence, for now, we can replace this with margin_large that is already present in the file, so as to avoid the repetition of dimens. This will be more uniform once that issue is fixed.